examples/jaeger-skaffold-trace/jaeger-all-in-one-template.yaml

```diff
@@ -0,0 +1,161 @@
+# Copyright 2019 The Skaffold Authors
```

Comment:
Copyright should be 2021 on a new file.  But if you copied this from elsewhere then we need to maintain their copyright too.

Reply:
In my PR, the example I used for jaeger is from:
https://github.com/jaegertracing/jaeger-kubernetes/blob/master/all-in-one/jaeger-all-in-one-template.yml
(with small modifications - `s/extensions\/v1beta1/apps\/v1` & `s/port: 80/port: port: 16686`)

It is also Apache 2 licensed.

I have added an attribution in a comment at the top of the yaml.  Is there any specific way of attributing the authors there that makes sense?  I wasn't sure if the linters checked this exact blurb at the top.

Reply:
I've updated this to have:
- 2021
- an attribution to the original yaml file it is based off of